advanced_source/dispatcher.rst

```diff
@@ -114,8 +117,8 @@ At this point, we have an operator with both CPU and CUDA implementations.  How
 can we add autograd support to it?  As you might guess, we will register an
 autograd kernel (similar to what's described in the `custom autograd function <cpp_autograd>`_ tutorial)!
 However, there is a twist: unlike the CPU and CUDA kernels, the autograd kernel
-needs to *redispatch*: it needs to call back into the dispatcher to get to
-the final CPU and CUDA implementations.
+usually has *redispatch*: it does proper backward node setup in autograd kernel,
+then calls back into the dispatcher to get to the CPU/CUDA inference implementations.
```

Comment:
The only thing about saying this here is that it doesn't actually match up to anything we see in the code sample below (because it's hidden away by `torch::autograd::Function`). I think it might be better to leave it as before (but following your lead, I'd change "get to the final CPU or CUDA implementations" to something like "get to the inference kernels, e.g. CPU or CUDA implementations" or similar).